### PR TITLE
feat(SubscribersHeader): add subscribers count

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -97,6 +97,8 @@
       "skipShapeProps": true
     }],
 
+    "react/jsx-closing-bracket-location": [1, {selfClosing: "after-props"}],
```
Comment (rosendi) [inline .eslintrc]: can we align it with the style:

```
"react/jsx-closing-bracket-location": [1, {
  "selfClosing": "after-props"}
],
```

File: client/routes/subscribers/components/SubscribersHeader.js [hunk 2]
```diff
@@ -58,7 +58,11 @@ const SubscribersHeader = ({ intl }) => (
 );
 
 SubscribersHeader.propTypes = {
+  count: PropTypes.shape({
```
Comment (rosendi) [inline client/routes/subscribers/components/SubscribersHeader.js]: can we switch  the order of `count`, `intl` to `intl`, new line, `count`?

File: .eslintrc (hunk 1 above)
Comment (zouxuoz) [inline .eslintrc]: fixed

File: client/routes/subscribers/components/SubscribersHeader.js (hunk 2 above)
Comment (zouxuoz) [inline client/routes/subscribers/components/SubscribersHeader.js]: fixed

**Pull request merged**